Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.cpp

```diff
@@ -343,13 +349,131 @@ namespace AzToolsFramework
+        }
+
+        // Increase Paintbrush Hardness
+        {
+            constexpr AZStd::string_view actionIdentifier = "o3de.action.paintBrushManipulator.increaseHardness";
+            AzToolsFramework::ActionProperties actionProperties;
+            actionProperties.m_name = PaintbrushIncreaseHardnessTitle;
+            actionProperties.m_description = PaintbrushIncreaseHardnessDesc;
+            actionProperties.m_category = "PaintBrush Manipulator";
+
+            actionManagerInterface->RegisterAction(
+                EditorMainWindowActionContextIdentifier,
+                actionIdentifier,
+                actionProperties,
+                []
+                {
+                    AdjustHardnessPercent(ed_paintBrushHardnessPercentAdjustAmount);
+                }
+            );
+
+            hotKeyManagerInterface->SetActionHotKey(actionIdentifier, "}");
+        }
+
+        // Decrease Paintbrush Size
+        {
+            constexpr AZStd::string_view actionIdentifier = "o3de.action.paintBrushManipulator.decreaseHardness";
+            AzToolsFramework::ActionProperties actionProperties;
+            actionProperties.m_name = PaintbrushDecreaseHardnessTitle;
+            actionProperties.m_description = PaintbrushDecreaseHardnessDesc;
+            actionProperties.m_category = "PaintBrush Manipulator";
+
+            actionManagerInterface->RegisterAction(
+                EditorMainWindowActionContextIdentifier,
+                actionIdentifier,
+                actionProperties,
+                []
+                {
+                    AdjustHardnessPercent(-ed_paintBrushHardnessPercentAdjustAmount);
+                }
+            );
+
+            hotKeyManagerInterface->SetActionHotKey(actionIdentifier, "{");
+        }
+    }
+
+    void PaintBrushManipulator::BindActionsToMode(AZStd::string_view modeIdentifier)
+    {
+        auto actionManagerInterface = AZ::Interface<AzToolsFramework::ActionManagerInterface>::Get();
+        AZ_Assert(actionManagerInterface, "PaintBrushManipulator - could not get ActionManagerInterface on RegisterActions.");
+
+        actionManagerInterface->AssignModeToAction(modeIdentifier, "o3de.action.paintBrushManipulator.increaseSize");
+        actionManagerInterface->AssignModeToAction(modeIdentifier, "o3de.action.paintBrushManipulator.decreaseSize");
+        actionManagerInterface->AssignModeToAction(modeIdentifier, "o3de.action.paintBrushManipulator.increaseHardness");
+        actionManagerInterface->AssignModeToAction(modeIdentifier, "o3de.action.paintBrushManipulator.decreaseHardness");
+    }
+
+    void PaintBrushManipulator::BindActionsToMenus()
+    {
+        auto menuManagerInterface = AZ::Interface<AzToolsFramework::MenuManagerInterface>::Get();
+        AZ_Assert(menuManagerInterface, "PaintBrushManipulator - could not get MenuManagerInterface on BindActionsToMenus.");
+
+        int baseSortKey = 6000; // arbitrary starting sort number
```

Comment:
Nit: Since you're not modifying the value, you could make it const or constexpr.